packages/fiori/src/SearchFieldTemplate.tsx

```diff
@@ -23,14 +31,14 @@ export default function SearchFieldTemplate(this: SearchField) {
 		) : (
 			<div class="ui5-search-field-root" role="search" onFocusOut={this._onFocusOutSearch}>
 				<div class="ui5-search-field-content">
-					{!!this.scopes.length &&
+					{!!this.getSlottedNodes<SearchScope>("scopes").length &&
```

Comment:
What is the reason the change the this.scopes to getSlottedNodes? I had a quick chat with Petar, there should be no problem here to use this.scopes?

Reply:
That used to be required when I was using SearchField inside the Search and having the slot set as a **slot** :)